contracts/interfaces/0.8.x/IFilteredSharedProjectMaxInvocationsLimit.sol

```diff
@@ -0,0 +1,30 @@
+// SPDX-License-Identifier: LGPL-3.0-only
+// Created By: Art Blocks Inc.
+
+pragma solidity ^0.8.0;
+
+/**
+ * @title This interface extends the IFilteredMinterV1 interface in order to
+ * add support for manually setting project max invocations.
+ * @author Art Blocks Inc.
+ */
+interface IFilteredSharedProjectMaxInvocationsLimit {
```

Comment:
Curious the rationale for having this be its own interface rather than just bundling into `IFilteredMinterV3`?

Reply:
yes this is definitely something I wasn't completely sure of, but the initial rationale is related to there being a few minters (polyptych namely if i recall correctly?) that don't/can't really utilize this project max invocations logic. will definitely revisit this as I flesh this out

Reply:
deleted this interface with the intention to consolidate in common minter interface